rllib/models/tf/__init__.py

```diff
@@ -0,0 +1,14 @@
+# TODO(sven): Add once ModelV1 is deprecated and we no longer cause circular
+#  dependencies b/c of that.
```

Comment:
Hmm what's the advantage of making this a sub-package?

Reply:
In general, everything should be a package, no? Looks nicer and allows for more intuitive imports.
Doing this always would enforce good class-hierarchy behavior. Currently, we have this circular dependency, which is why we cannot have this __init__.

Reply:
Fair enough.

Reply:
Cool.